app/styles/variables.css

```diff
@@ -19,4 +19,6 @@
   --color-green: #49a82e;
 
   --color-light-navyblue: #1d128340;
+
+  --color-cultured: #f6f6f6;
```

Comment:
I think we we can use variables which is prebuilt for instead of creating new one.

Reply:
No colour matching the background with pink as active color. That's why used the color from UX given

Reply:
Got it